test/unittests/parser_test.cpp

```diff
@@ -1461,5 +1456,4 @@ TEST(parser, milestone1)
         ElementsAre(Instr::local_get, 0, 0, 0, 0, Instr::local_get, 1, 0, 0, 0, Instr::i32_add,
             Instr::local_get, 2, 0, 0, 0, Instr::i32_add, Instr::local_tee, 2, 0, 0, 0,
             Instr::local_get, 0, 0, 0, 0, Instr::i32_add, Instr::end));
-    EXPECT_EQ(c.immediates.size(), 0);
```

Comment:
Actually it would have been nice to have the final "remove immediates from struct" as a separate commit to see the impact of changes.